src/views/SignupComponent.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
```

Comment:
Now since this is a container component let's name it as `SignupContainer.jsx` by the naming convention we adopted in the channel.

Reply:
Hello,
I have made the change